substrate/bin/node/runtime/src/impls.rs

```diff
@@ -236,6 +236,29 @@ impl InstanceFilter<RuntimeCall> for NftsCallFilter {
 	}
 }
 
+#[derive(
+	Clone, Eq, PartialEq, Encode, Decode, RuntimeDebug, MaxEncodedLen, scale_info::TypeInfo, Serialize, Deserialize,
+)]
+pub struct SenateAccountCallFilter;
+
+impl Contains<RuntimeCall> for SenateAccountCallFilter {
+	fn contains(c: &RuntimeCall) -> bool {
+		match c {
+			RuntimeCall::Utility(pallet_utility::Call::batch { calls }) => calls.iter().all(|inner_call| Self::contains(inner_call)),
+			RuntimeCall::Utility(pallet_utility::Call::batch_all { calls }) => calls.iter().all(|inner_call| Self::contains(inner_call)),
+			_ => matches!(c,
+				RuntimeCall::LLM(pallet_llm::Call::remark { .. }) |
+				RuntimeCall::LLM(pallet_llm::Call::send_llm { .. }) |
+				RuntimeCall::LLM(pallet_llm::Call::send_llm_to_politipool { .. }) |
+				RuntimeCall::Assets(pallet_assets::Call::transfer { .. }) |
+				RuntimeCall::Assets(pallet_assets::Call::transfer_keep_alive { .. }) |
```

Comment:
senate can also cancel scheduler and referenda.

Why are those calls not in this list but LLM remark is ?

Reply:
Because cancel scheduler and cancel referenda are direct calls made by Senate using pallet-collective Origin.
This is a call filter for Senate Custom Account, which allows executing stuff with Signed(SenateAccount) origin. So only spending stuff.